api/v1beta1/infraenv_types.go

```diff
@@ -88,6 +88,11 @@ type InfraEnvSpec struct {
 	// +kubebuilder:default=x86_64
 	// +optional
 	CpuArchitecture string `json:"cpuArchitecture,omitempty"`
+
+	// IPXEScriptType the script type that should be served (HostRedirect/Boot)
```

Comment:
Sorry, this is my last comment I think. :) Here we should have clear documentation about what each option does. And it should not assume that the user knows about the two different scripts already and what they do.